snapcraft/services/confdbschemas.py

```diff
@@ -65,6 +65,67 @@
     """
 )
 
+# --- Key Ordering and Custom Dumper ---
+# Define the desired order for keys within rules
+RULE_KEY_ORDER = ["request", "storage", "access", "summary", "content"]
+# Define the desired order for keys within views
+VIEW_KEY_ORDER = ["summary", "rules"]
+# Top level order (after base fields)
+TOP_LEVEL_ORDER = ["summary", "views", "body"]
```

Comment:
Where did the requirement for strict ordering come from and what does this have to do with adding a summary field? This is a massive amount of extra code to support an unknown requirement.